spec/javascripts/mobile/mobile_comments_spec.js

```diff
@@ -82,14 +82,8 @@ describe("Diaspora.Mobile.Comments", function(){
   describe("createComment", function () {
     beforeEach(function() {
       spec.loadFixture("aspects_index_mobile_post_with_comments");
-      var commentBoxHtml = spec.fixtureHtml("comments_mobile_commentbox");
       var link = $(".stream .comment-action").first();
       Diaspora.Mobile.Comments.showCommentBox(link);
-      jasmine.Ajax.requests.mostRecent().respondWith({
```

Comment:
@AugierLe42e You might also want to remove https://github.com/diaspora/diaspora/blob/develop/spec/controllers/jasmine_fixtures/comments_spec.rb then.